features/unit/rekey.feature

```diff
@@ -0,0 +1,28 @@
+@rekey
```

Comment:
Rename to `@unit.rekey`, then we can merge this in!

Reply:
This was causing some issues for SDK development so I went ahead updated the change.